### DISCUSSION TITLE
How to migrate usages of set(column: Column<*>, value: Any?) to set(column: Column<C>, value: C?) in 3.3?

### DISCUSSION THREAD

**Discussion by efenderbosch** [vincentlauvlwj/Ktorm#237] [Q&A] [answered]
Prior to 3.2 we were doing this:

```kotlin
it.columns(reified).forEach { (column, value) ->
    it[column.name] to value
}
```

That was deprecated in 3.2, so now we do this:
```kotlin
it.columns(reified).forEach { (column, value) ->
    set(column, value)
}
```

However, ``public fun set(column: Column<*>, value: Any?)`` has been deprecated. We don't have the type information here to be able to use the ``public fun <C : Any> set(column: Column<C>, value: C?)`` version. Any suggestions on what to do here?

**Comment by vincentlauvlwj** [comment, marked as answer]
You can cast `column` as `Column<Any>`: 

```kotlin
it.columns(reified).forEach { (column, value) ->
    set(column as Column<Any>, value)
}
```
